torchao/quantization/quant_api.py

```diff
@@ -660,23 +698,46 @@ def float8_dynamic_activation_float8_weight(
     if mm_config is None:
         mm_config = Float8MMConfig(use_fast_accum=True)
 
-    #TODO we are hardcoding TensorWise scaling, will follow up PR for Tensorwise scaling
+    activation_granularity, weight_granularity = _validate_granularity(granularity)
+
+    def get_block_size(x: torch.Tensor, granularity: _fp8_granularities):
```

Comment:
why is this not added to the official one in https://github.com/pytorch/ao/blob/144445a7a8e988059555421caafa17e0c1678053/torchao/quantization/observer.py#L92

Reply:
Ohh great call!